lib/associations/belongs-to-many.js

```diff
@@ -302,6 +304,71 @@ var BelongsToMany = function(source, target, options) {
 
 util.inherits(BelongsToMany, Association);
 
+BelongsToMany.prototype.get = function(instances, options) {
+  var throughModel = this.throughModel;
+
+  var throughOptions = _.assign({
+    where: _.set({}, this.foreignKey, {$in: _.map(instances, this.source.primaryKeyAttribute)}),
+    logging: options.logging
+  });
+
+  var association = this;
+
+  if(_.isUndefined(options.limit) && _.isUndefined(options.orderBy)) {
```

Comment:
Actually, we should have this for when the user specifically says include.order and not using the nested order syntax.